app/src/main/java/edu/usf/sas/pal/muser/manager/DeviceInformationManager.java

```diff
@@ -0,0 +1,79 @@
+package edu.usf.sas.pal.muser.manager;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.PowerManager;
+import android.view.accessibility.AccessibilityManager;
+
+import com.google.android.gms.common.GoogleApiAvailability;
+import com.simplecity.amp_library.ShuttleApplication;
+
+import edu.usf.sas.pal.muser.constants.EventConstants;
+import edu.usf.sas.pal.muser.model.DeviceInfo;
+import edu.usf.sas.pal.muser.model.Event;
+import edu.usf.sas.pal.muser.util.FirebaseIOUtils;
+import edu.usf.sas.pal.muser.util.PreferenceUtils;
+
+import static android.content.Context.ACCESSIBILITY_SERVICE;
+
+public class DeviceInformationManager {
+
+    private Context mApplicationContext;
```

Comment:
There is a general trend in Android for moving away from "Hungarian notation", or prefixing class member variables with `m`. Please refactor to `applicationContext` instead. Same for any other class member variable prefixed by `m`.